pkgs/by-name/li/libresample/package.nix

```diff
@@ -26,12 +29,13 @@ stdenv.mkDerivation rec {
 
   meta = {
     description = "Real-time library for sampling rate conversion library";
-    license = lib.licenses.lgpl2Plus;
-    homepage = "https://ccrma.stanford.edu/~jos/resample/Free_Resampling_Software.html";
+    homepage = "https://github.com/minorninth/libresample";
+    license = lib.licenses.bsd3; # OR LGPL-2.1-or-later
```

Comment:
We add or licenses as both

Reply:
As far as I can tell, this would make any software bill of materials for a Nix project using `libresample` strictly less useful, given that LGPL 2.1+’s requirements are a superset of BSD3’s, and there are examples of doing it the other way in‐tree (I suspect many dual licenses just aren’t represented at all, actually). However if you want me to make it less useful I can.

Reply:
Right now meta.licenses is always the collection of all licenses the project uses and we don't reflect the details. Anything else is done wrong.

Reply:
I’m not sure you understand how dual licensing works: any downstream can pick any licence, at their option. libresample can be used entirely under the terms of a BSD licence without any consideration of the LGPL terms. Therefore simply declaring BSD is entirely correct: Nixpkgs has the option to choose that licence, is exercising it, and neither Nixpkgs nor any package depending on this library is under any LGPL‐related obligations.

Consider a program that is available under either the GPL or a proprietary, non‐free licence. There are many historical examples of this, for instance Qt. Do you really think that it ought to be declared `licenses = [ lib.licenses.gpl3Only lib.licenses.unfreeRedistributable ]` and flag up unfree warnings, when the whole point is that anyone who receives the software can choose to use it under the terms of the GPL?